README.md

```diff
@@ -262,7 +264,158 @@ export default NuxtAuthHandler({
 })
 ```
 
-Note that there's way more options inside the `nextAuth.options` object, see [here](https://next-auth.js.org/configuration/options#options) for all available options.
+##### Example with a custom Strapi JWT provider 
+
+Here's what a full config can look like, wee allow authentication via:
+- Strapi v4 JWTs with the `CredentialsProvider`
+
+Note that the below implementation of the credentials provider stores the Strapi JWT in the profile field of the user. The user object will be wrapped inside the session. Sidenote: the session has it's own JWT. Sending the Nuxt-Auth session JWT token as bearer token will not work, you need to pass the token that is originally returned by Strapi to get authorised accessing any auth enabled resources on your Strapi Server.
```

Comment:
Can you please adapt the example to _not_ store the JWT inside the `profile` field of the user? AFAIU this is _not_ the right way to do it but rather a hack. So until we understand how to correctly expand the user object to store the JWT, I would rather just focus this section on how to do auth with strapi (which works without storing the strapi JWT).

Reply:
Tahis whole section can go, comment also unadressed